src/mmw/mmw/settings/production.py

```diff
@@ -67,33 +67,6 @@
     )
 })
 
-# Django Storages CONFIGURATION
```

Comment:
As [discussion](https://github.com/WikiWatershed/model-my-watershed/issues/3274#issuecomment-601923168) in the issue surfaced, this is dead code.